outlines/outline.py

```diff
@@ -0,0 +1,52 @@
+import ast
+from dataclasses import dataclass
+
+
+@dataclass
+class Outline:
+    """
+    Outline is a class that creates a callable object to generate responses
+    based on a given model and a prompt template.
+
+    Args:
```

Comment:
We've used the Numpy docstring style so far and should stick to it.

Reply:
What about having that checked by a pre-commit hook? https://github.com/dottxt-ai/outlines/pull/1368